src/shared/internal-calendar/internal-fixed-range-selection-calendar-day.tsx

```diff
@@ -0,0 +1,107 @@
+import dayjs, { Dayjs } from "dayjs";
+import { useMemo, useState } from "react";
+import { Text } from "../../text/text";
+import { CommonCalendarProps, View } from "./types";
+import { CalendarHelper } from "../../util/calendar-helper";
+import { HeaderCell, RowDayCell, Wrapper } from "./internal-calendar-day.style";
+import { FixedRangeDayCell } from "./fixed-range/fixed-range-cell";
+
+interface CalendarWeekSelectProps extends CommonCalendarProps {
+    selectedStartDate: string;
+    selectedEndDate: string;
+    calendarDate: Dayjs;
+    currentView: View;
+    onSelect: (value: Dayjs) => void;
+    onHover: (value: string) => void;
+    numberOfDays: number;
+}
+
+export const InternalFixedRangeSelectionCalendarDay = ({
```

Comment:
a more succinct name

```suggestion
export const FixedRangeCalendarDayView = ({
```